File: event/src/test/java/de/danoeh/antennapod/event/playback/SpeedChangedEventTest.java
package de.danoeh.antennapod.event.playback;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class SpeedChangedEventTest {

    @Test
    public void newSpeedIsStored() {
        SpeedChangedEvent event = new SpeedChangedEvent(1.5f);
        assertEquals(1.5f, event.getNewSpeed(), 0f);
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/playback/SpeedChangedEventTest.kt
package de.danoeh.antennapod.event.playback

import org.junit.Assert.assertEquals
import org.junit.Test

class SpeedChangedEventTest {

    @Test
    fun newSpeedIsStored() {
        val event = SpeedChangedEvent(1.5f)
        assertEquals(1.5f, event.newSpeed, 0f)
    }
}
